wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss

```diff
@@ -0,0 +1,147 @@
+@use '../abstracts' as *;
+
+.home {
+  &__about {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    color: $color-white-100;
+    background-color: $color-blue-000;
+    @include full-width-background($color-blue-000);
+
+    @include breakpoint('large') {
+      flex-wrap: nowrap;
+      gap: rem(32px);
+    }
+
+    &::before,
+    &::after {
+      @include z-index('postBG');
```

Comment:
Instead of using `postBG` here, it's better practice to add `homeAboutPONI` as an entry in the `z-index` map in the variables file. That way if changes need to be made to the `postBG` index, it doesn't have unexpected side effects on this part of the site where we wouldn't expect it.